dolphinscheduler-spi/src/main/java/org/apache/dolphinscheduler/spi/task/TaskExecutionContextCacheManager.java

```diff
@@ -66,4 +67,8 @@ public static boolean updateTaskExecutionContext(TaskRequest request) {
         taskRequestContextCache.computeIfPresent(request.getTaskInstanceId(), (k, v) -> request);
         return taskRequestContextCache.containsKey(request.getTaskInstanceId());
     }
+
+    public static Collection<TaskRequest> getAllTaskRequestList() {
```

Comment:
Wait, should we add some comment on this method?

Reply:
It is just a easy method without handle logic. So I think we can add comment but it is useless.